### PR TITLE
NPM publish testing (use the `-PdryRun=true` flag)

### PR DESCRIPTION
**Pull request by anton-bannykh**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -35,10 +89,24 @@ subprojects.stream().filter { it.parent != rootProject }.each { p ->
 
         dependencies {
             compile "org.jetbrains.kotlin:kotlin-stdlib-js:$kotlin_version"
+            compile 'org.apache.commons:commons-compress:1.14'
```
Comment (bashor) [inline build.gradle]: what for?

**Comment by bashor** [comment]
Kool! Thank you!

**Comment by anton-bannykh** [comment]
Oops, forgot to remove.

**Comment by anton-bannykh** [comment]
Removed unnecessary curly braces and the apache dependency

**Pull request merged**